### PR TITLE
[Doc] Add Node.js Client Docs

### PR DESCRIPTION
**Pull request by equanz**
## Motivation
Currently, There is no official document for [Node.js client](https://github.com/apache/pulsar-client-node).  
We want to find information about Node.js client from Official docs. 

## Modifications
Add Node.js client docs.

## Result
We can find information about Node.js client.

### CODE REVIEW

File: site2/docs/client-libraries-node.md [hunk 1]
```diff
@@ -0,0 +1,402 @@
...
+| `topic` | The Pulsar [topic](reference-terminology.md#topic) on which the consumer will establish a subscription and listen for messages. | |
+| `subscription` | The subscription name for this consumer. | |
+| `subscriptionType` | Available options are `Exclusive`, `Shared`, and `Failover`. | `Exclusive` |
+| `ackTimeoutMs` | Acknowledge timeout in milliseconds. | 0 |
+| `receiverQueueSize` | Sets the size of the consumer's receiver queue, i.e. the number of messages that can be accumulated by the consumer before the application calls `receive`. A value higher than the default of 1000 could increase consumer throughput, though at the expense of more memory utilization. | 1000 |
+| `receiverQueueSizeAcrossPartitions` | Set the max total receiver queue size across partitions. This setting will be used to reduce the receiver queue size for individual partitions if the total exceeds this value. | 50000 |
+| `consumerName` | The name of consumer. Currently(v2.4.1), [failover](concepts-messaging.md#failover) mode use consumer name in ordering. | |
+| `properties` | The metadata of consumer. | |
+
+### Consume example
```
Comment (Jennifer88huang) [inline site2/docs/client-libraries-node.md]: consumer example?

File: site2/docs/client-libraries-node.md [hunk 2]
```diff
@@ -0,0 +1,402 @@
...
+pulsar://pulsar.us-west.example.com:6650
+```
+
+If you're using [TLS encryption](security-tls-transport.md) or [TLS Authentication](security-tls-authentication.md), the URL will look like something like this:
+
+```http
+pulsar+ssl://pulsar.us-west.example.com:6651
+```
+
+## Creating a client
```
Comment (Jennifer88huang) [inline site2/docs/client-libraries-node.md]:
```suggestion
## Create a client
```

**Comment by equanz** [comment]
@Jennifer88huang 
As follow above, I fix some issues. I appreciate for your suggest.
If you notice more issue, please comment to me.

**Comment by equanz** [comment]
@nkurihar 
In [9161fc1](https://github.com/apache/pulsar/pull/5212/commits/9161fc1dfa37acf052e2d36c32ce4aa7ccbf0bf3), delete some description.
So, please review again.

**Comment by Jennifer88huang** [comment]
@equanz Thank you very much for your efforts.

**Comment by nkurihar** [comment]
run integration tests

**Comment by nkurihar** [comment]
run integration tests

**Pull request merged**